safe/gui/tools/test/test_multi_buffer.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+import os
+
+from qgis.core import QgsVectorLayer, QgsMapLayerRegistry
+from safe.datastore.folder import Folder
+from safe.definitions.fields import buffer_distance_field, hazard_class_field
+from safe.test.utilities import (
+    load_test_vector_layer,
+    standard_data_path,
+    get_qgis_app)
+from PyQt4 import QtGui
+
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+from safe.gui.tools.multi_buffer_dialog import (
+    MultiBufferDialog)
+
+output_path = standard_data_path(
+    'hazard', 'volcano_point_multi_buffer.geojson')
+data_store = Folder(os.path.dirname(output_path))
```

Comment:
I think it's better to have these vars in the `setUp`. You can add the `self.` prefix.